app-benchmarks/stress-ng/files/makefile.patch

```diff
@@ -0,0 +1,75 @@
+--- a/Makefile	2018-04-27 17:02:16.000000000 +0200
++++ b/Makefile	2018-05-05 00:15:00.687399505 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -DVERSION='"$(VERSION)"'
+ 
+ #
+ # Pedantic flags
+@@ -314,12 +314,10 @@
+ .o: stress-ng.h Makefile
+ 
+ .c.o: stress-ng.h Makefile $(SRC)
+-	@echo "CC $<"
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) $(LDFLAGS) -c -o $@ $<
```

Comment:
If it's rule for compiling object files, LDFLAGS are unnecessary. I suppose you may also remove it altogether and let make do its default (if it works).

Reply:
Seems to work, when I do remove it. Pushed now a new eBuild. I also set QA_WX_LOAD. Please see, if I should better remove it.